pkg/controller/plan/kubevirt.go

```diff
@@ -362,8 +362,6 @@ func (r *KubeVirt) vmImport(
 		object.Spec.Warm = true
 		if r.Migration.Spec.Cutover != nil {
```

Comment:
Please remove this nil check; now that the fallback to the Plan value is gone we need to set `object.Spec.FinalizeData = r.Migration.Spec.Cutover` in all cases so that the user can clear the field if they make a mistake.

```
	if r.Plan.Spec.Warm {
		object.Spec.Warm = true
		object.Spec.FinalizeDate = r.Migration.Spec.Cutover
		
	}
```